infrastructure/controller/pkg/endpoints/components_update.go

```diff
@@ -40,10 +40,8 @@ func (s *ComponentsUpdate) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	for _, component := range requestComponents.Components {
 		if !s.isComponentInstalled(component.Name) {
-			w.Header().Set("Content-Type", "application/json")
-			w.WriteHeader(http.StatusBadRequest)
-			w.Write([]byte(fmt.Sprintf(`{"error": "component %s is not installed"}`, component.Name)))
-			return
+			klog.Error("Trying to apply configuration for a component that is not installed: " + component.Name)
+			continue
```

Comment:
If there is a configuration in the `airy.yaml` file for at least one component that is not installed, the whole `airy config apply` command errors with a `Bad Request`. Therefore, I think that it is better if we just skip that component and at the end we return a list of components which have been updated.

Reply:
I agree. Small suggestion: Could you add a single sentence about this to the HTTP API docs?